### PR TITLE
Small doc and linting fixes

### PR DESCRIPTION
**Pull request by alexfikl**
Was looking through `mesh.generation` and found some typos, so went ahead and linted the whole thing. The commits should be fairly self-contained.

### CODE REVIEW

File: meshmode/mesh/processing.py [hunk 1]
```diff
@@ -1429,12 +1429,13 @@ def compute_new_map(old_map: AffineMap) -> AffineMap:
...
             fagrp_list = []
             for old_fagrp in old_fagrp_list:
-                if hasattr(old_fagrp, "aff_map"):
-                    new_fields = {"aff_map": compute_new_map(old_fagrp.aff_map)}
+                if isinstance(old_fagrp, InteriorAdjacencyGroup):
+                    new_fagrp: FacialAdjacencyGroup = replace(
+                        old_fagrp, aff_map=compute_new_map(old_fagrp.aff_map))
                 else:
-                    new_fields = {}
+                    new_fagrp = old_fagrp
```
Comment (inducer) [inline meshmode/mesh/processing.py:1436]:
```suggestion
                    assert not hasattr(old_fagrp, "aff_map")
                    new_fagrp = old_fagrp
```

File: meshmode/mesh/processing.py [hunk 2]
```diff
@@ -1429,12 +1429,13 @@ def compute_new_map(old_map: AffineMap) -> AffineMap:
...
 
-                fagrp_list.append(replace(old_fagrp, **new_fields))
+                fagrp_list.append(new_fagrp)
```
[the remark this answers is not in the document]
Comment (inducer) [reply meshmode/mesh/processing.py:1438]: I think it is. `InterPartAdjacencyGroup` appears to have an `aff_map`, too. I think the `assert` is also good to have, just to not make sure.

**Review by inducer — COMMENTED** [review]
LGTM, other than making the `replace` changes safer.

File: meshmode/mesh/processing.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexfikl) [reply meshmode/mesh/processing.py]: I'm not familiar with this chunk of code, so I'm not sure what you meant here. Is the assert enough or should this `isinstance` include the `InterPartAdjacencyGroup`?

[the remark this answers is not in the document]
Comment (inducer) [reply meshmode/mesh/processing.py]: Both. Fixed it.

**Pull request merged**